content/ct3a-guestbook.mdx

```diff
@@ -5,8 +5,6 @@ date: "2022-08-11"
 draft: false
 ---
 
-> Update: This blog post was written before [tRPC v10](https://trpc.io/docs/v10/) was released. You should use tRPC v10 now.
```

Comment:
Should probably add a line that this tutorial has been updated, ideally by the date above with `Updated on Oct 27, 2022`, etc. I'm not familiar with this repo so I don't know if `updated` is part of the blog post model.

Reply:
There's no updated field, a blockquote on saying its updated is a good addition imo.